target/linux/generic/hack-5.15/262-regmap-i2c-unhide.patch

```diff
@@ -0,0 +1,24 @@
+This makes it possible to select CONFIG_REGMAP and CONFIG_REGMAP_I2C.
+This is needed to compile this into the kernel and make use of it
+from out of tree drivers like hwmon-isl28022.
+
+--- a/drivers/base/regmap/Kconfig
++++ b/drivers/base/regmap/Kconfig
+@@ -6,7 +6,7 @@
+ config REGMAP
+ 	select IRQ_DOMAIN if REGMAP_IRQ
+ 	select MDIO_BUS if REGMAP_MDIO
+-	tristate
++	tristate "select REGMAP"
```

Comment:
Why do you need these as we already have regmap packaged as kmods?

Also, in the future please export the patches with `git format-patch` and include an SoB.

Reply:
Without unhiding these the driver doesn't compile, see forum discussion:
[how-to-set-dependency-in-kernel-module-package](https://forum.openwrt.org/t/how-to-set-dependency-in-kernel-module-package/166464)